pynetgear/__init__.py

```diff
@@ -283,14 +284,14 @@ def autodetect_url():
     return None
 
 
-def _find_node(response, service_name, xpath):
-    root = ET.fromstring(response)
-    namespace = {
-        "m": SERVICE_PREFIX + service_name,
-        "SOAP-ENV": "http://schemas.xmlsoap.org/soap/envelope/"
-    }
-
-    node = root.find(xpath, namespace)
+def _find_node(response, xpath):
+    it = ET.iterparse(StringIO(response))
+    # strip all namespaces
+    for _, el in it:
+        if '}' in el.tag:
+            el.tag = el.tag.split('}', 1)[1]
+    root = it.root
+    node = root.find(xpath)
```

Comment:
Please delete previous line and do it.root.find, the root var is never used afterwards.